### PR TITLE
[FIX] mail: mail tour fix

### PR DESCRIPTION
**Pull request by kp-merucore**
Current behavior before PR: 

The tour doesn't get past the first step

Desired behavior after PR is merged:

The tour will go to the next step

LINKS

PR https://github.com/odoo/odoo/pull/55876
Task-2313321

--
I confirm I have signed the CLA and read the PR guidelines at www.odoo.com/submit-pr

### CODE REVIEW

File: addons/mail/static/src/js/tour.js [hunk 1]
```diff
@@ -32,8 +32,8 @@ tour.register('mail_tour', {
     },
 }, {
     trigger: ".ui-menu-item > a",
-    auto: true,
-    in_modal: false,
+    content: _t("Create a public or private channel."),
```
Comment (seb-odoo) [inline addons/mail/static/src/js/tour.js:35]: I'm not expert in translations but I believe something has to be added in pot/po files when adding new strings in stable.

File: addons/mail/static/src/js/tour.js [hunk 2]
```diff
@@ -32,8 +32,8 @@ tour.register('mail_tour', {
     },
 }, {
     trigger: ".ui-menu-item > a",
-    auto: true,
```
Comment (seb-odoo) [inline addons/mail/static/src/js/tour.js:35]: What was the reason that `auto: true` didn't work?

Comment (hbh-odoo) [reply addons/mail/static/src/js/tour.js:35]: Yes! not working the step in manual onboarding tour so removed auto true and added the content and position for droplet

File: addons/mail/static/src/js/tour.js (hunk 1 above)
Comment (hbh-odoo) [reply addons/mail/static/src/js/tour.js]: Done!

**Comment by seb-odoo** [comment]
I'm not sure to understand properly the issue in v12 (I think this is not what is fixed here) and it is said in the issue it is fine to fix only in v13 so I'm closing this one.

**Pull request closed without merging**